packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -265,14 +266,20 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
     manager.setFocused(true);
 
     if (manager.focusedKey == null) {
+      let navigateToFirstKey = (key: Key | undefined, childFocus?: FocusStrategy) => {
+        if (key != null) {
+          manager.setFocusedKey(key, childFocus);
+          manager.replaceSelection(key);
```

Comment:
This should only happen if `selectOnFocus` is true right?

Reply:
good point, that's probably true